### DISCUSSION TITLE
为什么会不停扫描挂载的文件夹？

### DISCUSSION THREAD

**Discussion by gexiaopeng** [AlistGo/alist#6106] [General]
应用启动以后会不定期扫描挂载的文件夹，我的目录很多，这样会浪费资源。请问有设置可以关闭自动扫描吗？
![QQ截图20240227155224](https://github.com/alist-org/alist/assets/7857056/bad70740-36a3-4f0e-b230-71d4200b7b75)

**Comment by xhofe** [comment]
这是你的webdav客户端在扫描 非服务端主动扫描。

**Reply by gexiaopeng** [reply]
我是通过rclone  挂载了 alist的webdav，请问rclone为什么要扫描目录？ 可以关闭吗？
